CONTRIBUTING.md

```diff
@@ -530,6 +530,33 @@ Before continuing, please be sure to read the online documentation of pyscopg2 t
   - [Advanced parameter types](http://initd.org/psycopg/docs/usage.html#adaptation-of-python-values-to-sql-types)
 
 
+### Do not bypass the ORM
+
+You should never use the database cursor directly when the ORM can do the same thing! By doing so you are bypassing all the ORM features, possibly the transactions, access rights and so on.
+
+And chances are that you are also making the code harder to read and probably less secure (see also next guideline):
+
+```python
+# very very wrong
+cr.execute('select id from auction_lots where auction_id in (' +
+           ','.join(map(str, ids)) + ') and state=%s and obj_price>0',
+           ('draft',))
```

Comment:
If you don't have a constraint that valid explicit that the value must be a list of integer we are susceptible to sql injection with concatenate.

The sql injection is explain better [here](https://github.com/vauxoo-dev/maintainer-tools/blob/f2616656bb23b8242f3858c132c1812ff40fe0da/CONTRIBUTING.md#no-sql-injection)